src/_angular/file-selector/dtv-file-selector.js

```diff
@@ -204,6 +198,12 @@
                 scope.selectorValid = false;
               }
 
+              if (!scope.subscriptionOff && !scope.hideSubscription) {
+                // ensure subscription-status component does a refresh in case user subscribed from in-app storage
+                scope.isSubscribed = false;
```

Comment:
@stulees I checked that and the component is not showing anything when the user is subscribed. cheers